tests/integration/kubernetes/gha-run.sh

```diff
@@ -129,6 +129,10 @@ function deploy_kata() {
 	# Enable 'default_vcpus' hypervisor annotation
 	yq write -i "${tools_dir}/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml" 'spec.template.spec.containers[0].env[6].value' "default_vcpus"
 
+	if [ -n "${SNAPSHOTTER:-}" ]; then
```

Comment:
nit:
As SNAPSHOTTER is set as SNAPSHOTTER="${SNAPSHOTTER:-}" at the beginning of the file, you don't need to do this again here.
Just do the `if [ -n "${SNAPSHOTTER}" ]; then` and you'll be fine.

Reply:
Fixed.